fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -219,7 +219,7 @@ protected void buildCommonModel(PhysicalDataModel pdm, boolean fhirSchema, boole
             gen.buildSchema(pdm);
             switch (dbType) {
             case DB2:
-                gen.buildDatabaseSpecificArtifactsPostgres(pdm);
+                gen.buildDatabaseSpecificArtifactsDb2(pdm);
```

Comment:
I thought to limit the DB differences in the datautil/model layer, so I had added dbtype for the procedures, and then check dbtype before apply the objects.

Reply:
I totally agree with this change, just wanted to share my thoughts. :)

Reply:
it worked, I changed it and thought I could get away with doing what you did.  It didn't work so I had to split it out at this layer.